src/file_analysis/analyzer/x509/functions.bif

```diff
@@ -134,6 +134,54 @@ const EVP_MD* hash_to_evp(int hash)
 		}
 	}
 
+// Check a given hostname against a name given in a cert (SAN, CN) and
+// return if they match.
+bool check_hostname(std::string_view hostname, std::string_view certname)
```

Comment:
(I know you're testing this from script-land, but this functions seems like a nice candidate for doctest as well.)

Reply:
True - I didn't think of that. Is it ok that this one is just tested from scriptland?

Reply:
Yeah, that's fine.